### ISSUE TITLE
`--use-copy-link` breaks `GIT CLONE`

### ISSUE THREAD

**Issue by alexcb** [earthly/earthly#2272]
```Earthfile
VERSION --use-copy-link 0.6

FROM alpine/git:1.0.7
test:
    WORKDIR /test
    GIT CLONE https://github.com/moby/buildkit.git buildkit
    WORKDIR /test/buildkit
    RUN pwd
    RUN ls -la
    RUN ls -la .git # this fails
    RUN git status
    RUN git branch
    RUN test -f README.md
```

However if you remove the `--use-copy-link` feature flag (i.e. only use `VERSION 0.6`), it works.

**Comment by vladaionescu** [comment]
Strange... A simple solution might be to disable the copy link flag in the git clone. I think this call is the only thing used: https://github.com/earthly/earthly/blob/main/earthfile2llb/converter.go#L1280.

Though I'm not sure what the real cause of this is and whether it might impact other COPYs in subtle ways.

**Comment by alexcb** [comment]
it looks like it's being cloned to `/buildkit` and is ignoring the `WORKDIR`.

**Comment by alexcb** [comment]
fixed with #2273
